dynawo/sources/API/JOB/JOBJobEntry.h

```diff
@@ -97,6 +97,24 @@ class JobEntry {
    */
   const std::string& getName() const;
 
+  /**
+   * @brief Default constructor
+   */
+  JobEntry();
+
+  /**
+   * @brief Copy assignment operator
+   * @param other the job entry to copy
+   * @returns reference to this
+   */
+  JobEntry& operator=(const JobEntry& other);
+
+  /**
+   * @brief Copy constructor
+   * @param other the job entry to copy
+   */
+  JobEntry(const JobEntry& other);
+
  private:
   boost::shared_ptr<ModelerEntry> modelerEntry_;        ///< Modeler entries container
```

Comment:
the outputsEntry is missing the copy and assignement operator if I'm not mistaken

Reply:
OK